### PR TITLE
[PAPI-371] Ethereum address validation and checksum

### PR DESCRIPTION
**Pull request by drmathias**
Validates an Ethereum address of a wrapped Ethereum and supports the following:
* Null token address, for the base token (ETH)
* Ethereum address without a hex prefix
* Unformatted ethereum address with hex prefix
* Checksum ethereum address

Also formats Ethereum addresses to the checksum variant with the hex prefix (0x)

---

Local example:

```
{
    "address": "PEDF7x2YNDfiHsfSiLZ3qx3dAJ3eNbAzFf",
    "name": "Uniswap (InterFlux)",
    "symbol": "iUNI",
    "decimals": 18,
    "sats": 1000000000000000000,
    "totalSupply": "1000000000.000000000000000000",
    "attributes": [
        "NonProvisional",
        "Interflux"
    ],
    "wrappedToken": {
        "custodian": "PBHvTPaLKo5cVYBFdTfTgtjqfybLMJJ8W5",
        "chain": "Ethereum",
        "address": "0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984",
        "trusted": true,
        "validated": true,
        "createdBlock": 66296,
        "modifiedBlock": 66296
    },
    "createdBlock": 66296,
    "modifiedBlock": 66296
}
```

### CODE REVIEW

File: src/Opdex.Platform.Infrastructure/Data/Handlers/Tokens/Wrapped/PersistTokenWrappedCommandHandler.cs [hunk 1]
```diff
@@ -20,13 +20,15 @@ public class PersistTokenWrappedCommandHandler : IRequestHandler<PersistTokenWra
                 {nameof(TokenWrappedEntity.Owner)},
                 {nameof(TokenWrappedEntity.NativeChainTypeId)},
                 {nameof(TokenWrappedEntity.NativeAddress)},
+s                {nameof(TokenWrappedEntity.Validated)},
```
Comment (mrtpain) [inline src/Opdex.Platform.Infrastructure/Data/Handlers/Tokens/Wrapped/PersistTokenWrappedCommandHandler.cs:23]: rogue "s" at the beginning of the line.

Comment (drmathias) [reply src/Opdex.Platform.Infrastructure/Data/Handlers/Tokens/Wrapped/PersistTokenWrappedCommandHandler.cs]: Removed

**Review by mrtpain — APPROVED** [review]
LGTM, nice work, will be interesting testing this out with a valid but all lowercase address from Etherscan. Did you happen to give that a shot?

**Comment by drmathias** [comment]
> LGTM, nice work, will be interesting testing this out with a valid but all lowercase address from Etherscan. Did you happen to give that a shot? 

Yeah, tried it out and also added tests for formatting the Ethereum address

**Pull request merged**